$NetBSD: patch-aa,v 1.1 2007/01/12 14:12:17 rillig Exp $

Solaris is kind of UNIX-like, but does not have the cfmakeraw function.

--- console.c.orig	2003-01-20 14:53:08.000000000 +0100
+++ console.c	2007-01-12 15:06:02.202678144 +0100
@@ -7,6 +7,7 @@
 #include <stdio.h>
 #include <stdarg.h>
 #include <stdlib.h>
+#include <string.h>	/* Solaris requires this for FD_ZERO() */
 
 #ifdef DOS
  #include <conio.h>
@@ -33,7 +34,7 @@ c_refresh(void)
 void
 c_init(void)
 {
-#ifdef UNIX
+#if defined(UNIX) && !defined(__sun)
  struct termios t;
 
  console_ok=0;
